openwisp_radius/base/models.py

```diff
@@ -142,6 +143,9 @@
 STRATEGIES = (('prefix', _('Generate from prefix')), ('csv', _('Import from CSV')))
 
 _NOT_BLANK_MESSAGE = _('This field cannot be blank.')
+GET_IP_LIST_HELP_TEXT = _(
```

Comment:
let's flag this as private, I would also do the same for `STRATEGIES` above:

`_GET_IP_LIST_HELP_TEXT` and `_STRATEGIES`